content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
 | Finish          |       3 |       4 | FT, FTN, FTV, F2V    |
 | Situation       |     2/0 |     4/0 | SIT                  |
 
+Question Type Minimums
+: For a given Question Type, the minimum number of questions must be asked in Questions Numbered 1-20. This excludes any questions appended with "A" or "B."
+
+Question Type Maximums
+: For a given Question Type, the maximum number of questions must not be exceeded in all "non-Overtime" questions. This includes any questions appended with "A" or "B."
+
+Overtime Specifics
+Question Type maximums can be exceeded in Overtime.
+
+## Question Writing
+
+### Preparation of Questions
+All questions must contain verbatim material. On some types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
+
+### Question Types
+
+#### Interrogative
+Should not be overly long and should be clear and precise.
+Must have exactly one inserted interrogative word.
+Must have a single answer.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+The Question Type abbreviation is "INT"
```

Comment:
I also decided to define the default Type abbreviation.

Reply:
We'll want to put the abbreviations in a separate document from the main rulebook of course, but I agree with standardizing them.

Reply:
Standardizing, absolutely yes. Required. Separate document, no. The rulebook is the base document. You shouldn't have to reference another document to understand the rulebook. It's not overly burdensome to include abbreviations. We can easily add a list, table, or just a section of terms and definitions. But we could also use the Associated Press style:

> Interogative (INT) or "standard" questions are...

Reply:
I've generally put Definitions in the area where they're "needed."

And when I used a defined term, I try to capitalize it.

Would love ideas on:
(1) placement of definitions
(2) how to denote you're using a word/term that is defined elsewhere

Reply:
AP style says if it's an acronym, you define it on first use, then always use the acronym elsewhere. Thus, if you see an acronym you don't know, you just scan upward until you find its first usage.

I'm not suggesting that's what we do universally, but it's typically a lot easier and flows nicer than having a definitions section. Also, this only works for acronyms in AP. We could use the same idea with phrases, though. But we'd need some sort of markup style for it, I would think.

As a convention, we could say that any terms (acronyms, words, or phrases) that have definitions could always appear in italic, and first usage defines them. So for example:

> *Context* refers to blah, blah, blah. Interrogative (*INT*) questions, also referred to as "standard" questions, have a *context* of 5 verses above and below the first and last verses of content that forms the *INT*.

That's a fair bit of text OCD, but if we're consistent with it, it could be really useful.

Reply:
Massively useful.

I like all your notation proposals.

Can we skip those for now, and hit them with fixup PRs? Basically concentrate on content now, and then visual cleanup next?

Reply:
Yeah. I was planning to do OCD passes on every MR anyway. Well, after I merge them, I'll push an OCD commit.

Reply:
I am happy to do that for mine first. It's just so much easier to have a single focus on each MR.